src/utils/compressImages.ts

```diff
@@ -25,13 +25,15 @@ const compressImage = ({
 
       const originWidth = image.width;
       const originHeight = image.height;
-      const widthRatio = originWidth / (resizingWidth || originWidth);
-      const heightRatio = originHeight / (resizingHeight || originHeight);
+      let targetResizingWidth = (!resizingWidth || resizingWidth > originWidth) ? originWidth : resizingWidth;
+      let targetResizingHeight = (!resizingHeight || resizingHeight > originHeight) ? originHeight : resizingHeight;
```

Comment:
```suggestion
      const targetResizingWidth = (!resizingWidth || resizingWidth > originWidth) ? originWidth : resizingWidth;
      const targetResizingHeight = (!resizingHeight || resizingHeight > originHeight) ? originHeight : resizingHeight;
```
I think it's okay to keep them as const since they won't be re-assigned.

Reply:
They have a chance of being re-assigned on the line 38 and 40 😁

Reply:
Ahh right. I missed that.